<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_ingresarProcesos">
          <property name="title">
           <string>Ingresar Procesos</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Numero de Procesos</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_numeroProcesos"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_agregarProcesos">
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_listaProcesos">
          <property name="title">
           <string>Lista de Procesos</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_listaProcesos"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Procesos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Estatus</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_lotesPendientes">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Lotes Pendientes</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_iniciar">
             <property name="text">
              <string>Iniciar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Contador Global</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdNumber"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Proceso en ejecuci贸n</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_procesoEjecucion">
             <property name="rowCount">
              <number>5</number>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <row>
              <property name="text">
               <string>Programa</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Operaci贸n</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Estimado</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Transcurrido</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Restante</string>
              </property>
             </row>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Procesos terminados</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_procesosTerminados">
             <property name="columnCount">
              <number>5</number>
             </property>
             <column>
              <property name="text">
               <string>Programa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Primer Dato</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Operaci贸n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Segundo Dato</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Resultado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Lote en ejecuci贸n</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_loteEjecucion">
             <property name="rowCount">
              <number>0</number>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Programa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Tiempo Estimado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton_agregarProcesos</tabstop>
  <tabstop>lineEdit_lotesPendientes</tabstop>
  <tabstop>tableWidget_loteEjecucion</tabstop>
  <tabstop>tableWidget_procesoEjecucion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
